cli/src/config.rs

```diff
@@ -139,14 +141,9 @@ impl Configuration {
     /// - Configuration `TrustedPeers` contains entries with duplicate public keys
     pub fn load_environment(&mut self) -> Result<()> {
         iroha_config::Configurable::load_environment(self)?;
-        self.finalize();
+        self.finalize()?;
         Ok(())
     }
-
-    /// Get `public_key` and `private_key` configuration parameters.
-    pub fn key_pair(&self) -> iroha_crypto::KeyPair {
-        iroha_crypto::KeyPair::new(self.public_key.clone(), self.private_key.clone())
-    }
```

Comment:
do you think this method is useful in the public API?

Reply:
Adding it back in should be easy.